### PR TITLE
f-strings instead of str.format()

### LINKED ISSUE #69: Replace str.format() with f-strings where possible
re: https://github.com/r5py/r5py/pull/61#issuecomment-1125052362

### PR DESCRIPTION
**Pull request by christophfink**
Replaced all occurrences of str.format() with the faster, neater, and shorter f-strings.

Closes #69

### CODE REVIEW

File: src/r5py/util/validatingrequestssession.py [hunk 1]
```diff
@@ -54,9 +54,7 @@ def request(self, method, url, checksum, **kwargs):
 
         if digest != checksum:
             raise ChecksumFailed(
-                ("Checksum failed for {}" + "expected {}, got {}").format(
-                    url, checksum, digest
-                )
+                f"Checksum failed for {url}" + "expected {checksum}, got {digest}"
             )
```
Comment (HTenkanen) [inline src/r5py/util/validatingrequestssession.py:58]: This won't produce expected text. The correct syntax would be:
```python
f"Checksum failed for {url}, expected {checksum}, got {digest}."
```

**Review by HTenkanen — CHANGES_REQUESTED** [review]
Looks good! Found one f-string that doesn't work as expected, added suggestion for the correct syntax.

File: src/r5py/util/validatingrequestssession.py (hunk 1 above)
Comment (christophfink) [reply src/r5py/util/validatingrequestssession.py:58]: Good spot! How did I miss this?

**Review by HTenkanen — APPROVED** [review]
Looks good!

**Pull request merged**